deno.json

```
@@ -0,0 +1,39 @@
+{
+  "tasks": {
+    "test": "deno test --allow-all --unstable --unstable-kv ./packages/*/__tests__/deno/*.ts"
+  },
+  "workspace": [
+    "./packages/denokv",
+    "./packages/deta",
+    "./packages/file",
+    "./packages/free",
+    "./packages/mongodb",
+    "./packages/psql",
+    "./packages/redis",
+    "./packages/supabase",
+    "./packages/pocketbase",
+    "./libs/utils"
+  ],
+  "exclude": [
+    "./**/examples/*.ts"
+  ],
+  "fmt": {
+    "useTabs": false,
+    "lineWidth": 100,
+    "indentWidth": 2,
+    "semiColons": true,
+    "singleQuote": true,
+    "proseWrap": "preserve",
```

Some are the default values so you might want to skip them.
```suggestion
    "lineWidth": 100,
    "singleQuote": true,
    "proseWrap": "preserve",
```